.github/workflows/labels.yml

```diff
@@ -0,0 +1,164 @@
+                    black garak
+                    \`\`\`
+
+                    Please update your branch after these have been made, and reach out if you have any problems.
+                  `
+                },
+                'needs-unique-branch': {
+                  close: true,
+                  comment: `
+                    Thanks for your pull request! We require for all contributed code to come from a **from a unique branch** in your repository before it can be merged.
+
+                    Please create a new branch in your fork of garak and resubmit this from that branch.
+
+                    If you are using Git on the command line that may look like:
+
+                    \`\`\`
+                    # Checkout the main branch
+                    git checkout main
+
+                    # Create a new branch for your feature
+                    git checkout -b <BRANCH_NAME>
+
+                    # Add your new files
+                    git add plugins/my-cool-new-plugin
+
+                    # Commit your changes with a relevant message
+                    git commit
+
+                    # Push your changes to GitHub
+                    git push origin <BRANCH_NAME>
+
+                    # Now browse to the following URL and create your pull request!
+                    # - https://github.com/leondz/garak/pulls
+                    \`\`\`
+
+                    This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.
+
+                    Please do resubmit from a unique branch, we greatly value your contribution! :tada:
+                  `
+                },
+                'needs-testing-environment': {
+                  close: false,
+                  comment: `
+                    Thanks for your pull request! As part of our landing process, we manually verify that all plugins work as expected.
+
+                    We have been unable to test this plugin successfully. This may be due to software or hardware requirements we cannot replicate.
+
+                    To help unblock this pull request, please:
+
+                    - Comment with links to documentation on how to set up an environment, and provide exact software version numbers to use
+                    - Or comment guided steps on how to set up our environment for testing this plugin
+
+                    Once there's a clear path for testing and evaluating this code, we can progress with this further.
+                  `
+                },
+                'needs-pull-request-template': {
+                  close: false,
+                  comment: `
+                    When creating a pull request, please ensure that the default pull request template has been updated with the required details.
```

Comment:
how do we balance docs on our PRs? we want to set a good example, but we also trust each other more and sustain a high velocity

Reply:
Since this action is `manual` based on collaborators adding tags this will not really change much. It is just a nicer way or asking the contributor to take action as the project vs as the single collaborator.